x/concentrated-liquidity/lp_test.go

```diff
@@ -24,19 +24,17 @@ func (s *KeeperTestSuite) TestCreatePosition() {
 	denom0 := "eth"
 	denom1 := "usdc"
 
-	amount0Desired := sdk.OneInt()
-	amount1Desired := sdk.NewInt(5000)
-	expectedLiquidityCreated := sdk.MustNewDecFromStr("1517.882323")
+	amount0Desired := sdk.NewInt(1000000)
+	amount1Desired := sdk.NewInt(5000000000)
 
 	s.SetupTest()
 
 	s.App.ConcentratedLiquidityKeeper.CreateNewConcentratedLiquidityPool(s.Ctx, poolId, denom0, denom1, currentSqrtP, currentTick)
 
-	asset0, asset1, liquidityCreated, err := s.App.ConcentratedLiquidityKeeper.CreatePosition(s.Ctx, poolId, s.TestAccs[0], amount0Desired, amount1Desired, sdk.OneInt(), sdk.OneInt(), lowerTick, upperTick)
+	asset0, asset1, err := s.App.ConcentratedLiquidityKeeper.CreatePosition(s.Ctx, poolId, s.TestAccs[0], amount0Desired, amount1Desired, sdk.ZeroInt(), sdk.ZeroInt(), lowerTick, upperTick)
 	s.Require().NoError(err)
-	s.Require().Equal(amount0Desired, asset0)
-	s.Require().Equal(amount1Desired, asset1)
-	s.Require().Equal(expectedLiquidityCreated, liquidityCreated)
+	s.Require().Equal(amount0Desired, asset0.Int64())
+	s.Require().Equal(amount1Desired, asset1.Int64())
```

Comment:
Can we compare these as strings instead of ints, please? When we convert these to `int64`, and continue to extending the tests, they often break due to overflows